pa_config/rca.conf

```diff
@@ -58,19 +58,19 @@
   },
 
   "muted-rcas": [
-    "HotShardRca",
-    "HotShardClusterRca",
-    "HeapAllocRateByShardTemperatureCalculator",
-    "HeapAllocRateByShardAvgTemperatureCalculator",
-    "HeapAllocRateShardIndependentTemperatureCalculator",
-    "HeapAllocRateTotalTemperatureCalculator",
-    "CpuUtilByShardsMetricBasedTemperatureCalculator",
-    "AvgCpuUtilByShardsMetricBasedTemperatureCalculator",
-    "ShardIndependentTemperatureCalculatorCpuUtilMetric",
-    "TotalCpuUtilForTotalNodeMetric",
-    "CpuUtilDimensionTemperatureRca",
-    "HeapAllocRateTemperatureRca",
-    "NodeTemperatureRca",
-    "ClusterTemperatureRca"
-   ]
+  //   "HotShardRca",
+  //   "HotShardClusterRca",
+  //   "HeapAllocRateByShardTemperatureCalculator",
+  //   "HeapAllocRateByShardAvgTemperatureCalculator",
+  //   "HeapAllocRateShardIndependentTemperatureCalculator",
+  //   "HeapAllocRateTotalTemperatureCalculator",
+  //   "CpuUtilByShardsMetricBasedTemperatureCalculator",
+  //   "AvgCpuUtilByShardsMetricBasedTemperatureCalculator",
+  //   "ShardIndependentTemperatureCalculatorCpuUtilMetric",
+  //   "TotalCpuUtilForTotalNodeMetric",
+  //   "CpuUtilDimensionTemperatureRca",
+  //   "HeapAllocRateTemperatureRca",
+  //   "NodeTemperatureRca",
+  //   "ClusterTemperatureRca"
```

Comment:
Any reason we are removing these from Muted RCA List?

Reply:
I was a mistake to keep it in the PR. I only did it to test the changes.